Embeddings upgrade

Markek1 opened the pull request.

Markek1 committed:
Added ./training/data volume in alphafind-api

Markek1 committed:
Changed pipeline to work with 3d-af-surfer embeddings

ProchazkaDavid commented on training/alphafind_training/create_embedding_old.py:
This file is not needed and should be removed.

ProchazkaDavid reviewed (changes requested):
Please remove the mentioned file and fix the CI pipeline 🙂

Markek1 committed:
Deleted old code and fixed pipeline

ProchazkaDavid committed:
Add installation of libhdf5-dev to Dockerfile

- Fixes "error: libhdf5.so: cannot open shared object file: No such file or director"

ProchazkaDavid commented:
The broken pipeline is due to another pull request. The functionality of this PR has been tested on Linux by @Markek1.
